components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAOImpl.java

```diff
@@ -424,6 +424,43 @@ public void revokeOAuthConsentByApplicationAndUser(String username, String tenan
         }
     }
 
+    /**
+     * Revoke the OAuth consents which is recorded in the IDN_OPENID_USER_RPS table against the Application
```

Comment:
```suggestion
     * Revoke the OAuth consents which is recorded in the IDN_OPENID_USER_RPS table against the application.
```

Reply:
Addressed in [6918b82](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/6918b828f239ba766e20f66e6d0af997d891413a)